hw/top_earlgrey/ip/xbar_main/dv/autogen/xbar_main_sim.core

```diff
@@ -12,8 +12,8 @@ filesets:
       - lowrisc:ip:xbar_main
       - lowrisc:dv:dv_utils
       - lowrisc:dv:xbar_tb
+      - lowrisc:dv:xbar_main_bind
     files:
-      - xbar_main_bind.sv
```

Comment:
Separated this out into its own core file since this is needed at the chip level.

Reply:
this file is autogenerated. I will make another PR to update the template of this file and make xbar_*_bind.core auto-generated, when this PR is merged.

Reply:
Already done in this PR.